react/components/Footer/global.css

```diff
@@ -40,8 +40,8 @@
 }
 
 .vtex-footer__payment-form-item {
-  height: 64px;
-  width: 64px;
+  height: 2.533rem;
+  width: 2.533rem;
 }
 .vtex-footer__vtexlogo-form-item {
   height: 64px;
```

Comment:
Use the Tachyons class ```h3```.

Reply:
Isn't `h3` class setting the height for `4rem`? 2.533 was a design's requirement

Reply:
Could be .h2 { height: 2rem; }